### ISSUE TITLE
This effects both ST & STB: Not able to play archived content.

### ISSUE THREAD

**Issue by ger2github** [yuliskov/SmartTube#2726]
All apps; STB, ST, Youtube, and browrsers are running on Nvidia (2019) Shield Pro all current updates installed.

Several day ago after update, STB start having issues with playing content more than a couple of months old, not all, but some; to address this I would switch to ST, annoying but workable.

Over the last week did update to STB, all older content stop playing. Very annoying! 

Switched to ST. This Saturday ST updated, and now has the same issues!

I the switched to Youtube for Android TV, none of these issues, it works as it should. Using browser at youtube.com, this also works.

Concullsion: The lastest changes made to both STB and ST are corrupted at the same point. Also; you need to provide a way to rollback update on both STB and ST.

Finally, the QR Code for the email link points to firsthash@gmail.com in both STB and ST. Is this right? (as email keeps boncing)

**Comment by kokofixcomputers** [comment]
> Concullsion: The lastest changes made to both STB and ST are corrupted at the same point. Also; you need to provide a way to rollback update on both STB and ST.

No, it is all because of Google (owner of YouTube) not liking you to use an unofficial app.
